tests/Unit/HSAIL/activelaneid.cpp

```diff
@@ -1,4 +1,4 @@
-// XFAIL: Linux
+// XFAIL: *
```

Comment:
this would make this test fail even on HSAIL backend. it's better to use __hcc_backend__ macro to check. I'll fix it. the same goes to activelanepermute right below.